src/main/java/uk/ac/ebi/eva/evaseqcol/entities/SeqColExtendedDataEntity.java

```diff
@@ -52,7 +51,7 @@ public class SeqColExtendedDataEntity<T> {
     private SeqColEntity.NamingConvention namingConvention;
 
     public enum AttributeType {
-        names, sequences, md5DigestsOfSequences, lengths, sortedNameLengthPairs
+        names, sequences, md5_sequences, lengths, sorted_name_length_pairs
```

Comment:
I think it's because we need the string forms of these to equal the property names in the JSON for the comparison endpoint... However I agree it looks weird in the Java, if this is the only reason maybe we could use one of the solutions [here](https://stackoverflow.com/questions/604424/how-to-get-an-enum-value-from-a-string-value-in-java) to have different string & identifier values.

Reply:
Yes, this was the reason for the change : 
> I think it's because we need the string forms of these to equal the property names in the JSON for the comparison endpoint... 

However I agree it's not a good solution. 
**April**'s proposal was ideal. I just commited the change.